include/IECore/Timer.h

```diff
@@ -92,13 +93,25 @@ class IECORE_API Timer
 
 class IECORE_API ScopedTimer
 {
+	typedef std::string Channel;
```

Comment:
Can we drop this whole commit from the PR? Its unrelated and has its own PR with review comments already.

Reply:
Yes sorry, I had edited the description of the PR after that initial email got sent. It's just here still in case one of your comments requires me to do some profiling.